cms/static/js/views/manage_users_and_roles.js

```diff
@@ -0,0 +1,306 @@
+/*
+    Code for editing users and assigning roles within a course team context.
+*/
+define(['jquery', 'underscore', 'gettext', "js/views/baseview", 'js/views/feedback_prompt', 'js/views/utils/view_utils'],
+function($, _, gettext, BaseView, PromptView, ViewUtils) {
+    'use strict';
+    var default_messages = {
+        defaults: {
+            confirmation: gettext("Ok"),
+            changeRoleError: gettext("There was an error changing the user's role"),
+            unknown: gettext('Unknown')
+        },
+        errors: {
+            addUser: gettext('Error adding user'),
+            deleteUser: gettext('Error removing user')
+        },
+        invalidEmail: {
+            title: gettext('A valid email address is required'),
+            message: gettext('You must enter a valid email address in order to add a new team member'),
+            primaryAction: gettext('Return and add email address')
+        },
+        alreadyMember: {
+            title: gettext('Already a course team member'),
```

Comment:
I found these "course team" specific messages confusing in this general file. You override it for both course teams and library teams, so perhaps don't supply it here (or make the text generic).